app/feature_defaults_unittest.cc

```diff
@@ -136,7 +136,6 @@ TEST(FeatureDefaultsTest, DisabledFeatures) {
       &extensions_features::kExtensionsManifestV3Only,
       &features::kToolbarPinning,
 #endif
-      &features::kAttributionReportingCrossAppWebOverride,
```

Comment:
is attribution reporting disabled in general on Brave?

Reply:
Attribution reporting should be disabled via Privacy Sandbox settings override in https://github.com/brave/brave-core/blob/master/components/privacy_sandbox/brave_privacy_sandbox_settings.h/cc